front/app/containers/Admin/settings/registration/CustomFieldSettings/index.tsx

```diff
@@ -70,285 +68,196 @@ const StyledIconTooltip = styled(IconTooltip)`
+                    {isBuiltInField(field) && (
+                      <div>
+                        <FormattedMessage
+                          {...customfieldMessages.defaultField}
+                        />
+                      </div>
                     )}
-                  </TextCellContent>
-                  {field.attributes.required && (
-                    <StyledBadge className="inverse" color={colors.error}>
-                      <FormattedMessage {...customfieldMessages.required} />
-                    </StyledBadge>
-                  )}
-                </StyledTextCell>
-                {isBuiltInField(field) && (
-                  <div>
-                    <FormattedMessage {...customfieldMessages.defaultField} />
-                  </div>
-                )}
-                {isHiddenField(field) && (
-                  <div>
-                    <FormattedMessage {...customfieldMessages.hiddenField} />
-                  </div>
-                )}
-                <Buttons>
-                  {!isBuiltInField(field) && !isHiddenField(field) && (
-                    <Button
-                      className={`e2e-delete-custom-field-btn e2e-${field.attributes.title_multiloc['en-GB']}`}
-                      onClick={this.handleOnDeleteClick(field.id)}
-                      buttonStyle="text"
-                      icon="delete"
-                    >
-                      <FormattedMessage
-                        {...customfieldMessages.deleteButtonLabel}
-                      />
-                    </Button>
-                  )}
-                  {!isHiddenField(field) && (
-                    <Button
-                      className={`e2e-custom-field-edit-btn e2e-${field.attributes.title_multiloc['en-GB']}`}
-                      linkTo={`/admin/settings/registration/custom-fields/${field.id}/field-settings`}
-                      buttonStyle="secondary"
-                      icon="edit"
-                    >
-                      <FormattedMessage
-                        {...customfieldMessages.editButtonLabel}
-                      />
-                    </Button>
-                  )}
-                </Buttons>
-              </SortableRow>
-            );
-          })}
-        </CustomFieldsList>
-        <Button
-          buttonStyle="cl-blue"
-          icon="plus-circle"
-          linkTo="/admin/settings/registration/custom-fields/new"
-        >
-          <FormattedMessage {...messages.addAFieldButton} />
-        </Button>
-      </Section>
-    );
-  }
-}
-
-const CustomFieldsListWithHoCs = injectIntl(CustomFields);
-
-export default () => {
-  const { data: userCustomFields } = useUserCustomFields();
-  const { mutateAsync: deleteUserCustomField } = useDeleteUserCustomField();
-  const { mutateAsync: reorderCustomFieldForUsers } =
-    useReorderUserCustomField();
-  const { mutateAsync: updateCustomFieldForUsers } = useUpdateUserCustomField();
```

Comment:
Changed to mutate instead of mutateAsync as [recommended](https://tkdodo.eu/blog/mastering-mutations-in-react-query#mutate-or-mutateasync).